package.json

```diff
@@ -40,6 +40,7 @@
     "fibers": "https://github.com/icenium/node-fibers/tarball/v1.0.6.3",
     "filesize": "3.1.2",
     "gaze": "0.5.2",
+    "glob": "^7.0.3",
     "gulp": "3.9.0",
```

Comment:
I have absolutely no idea :D Lets do a search.
vinylFilterSince may be dropped as well.